GOVERNANCE.md

```diff
@@ -0,0 +1,115 @@
+
+## Owners Structure
+
+There are two types of owners in the Ratify project organized hierarchically. Ratify [org owners][ratify-project-owners] oversee the overall project and its health. Subproject owners focus on a single repository, a group of related repositories, a service (e.g., a website), or subproject to support the other subprojects (e.g., marketing or community management). 
+
+Changes in Ratify Org owners have to be announced via an [ratify-project/community/issues][ratify-issues-new]. Changes to sub-project owners are to be announced via the appropriate sub-project issue.
+
+### Owner Responsibility
+
+Ratify owners adhere to the requirements and responsibilities set forth in the respective [Ratify Org Owners](#ratify-org-owners) and [Subproject Owners](#subproject-owners). They further pledge the following:
+
+- To act in the best interest of the project and subprojects
+- To ensure that project and subproject development and direction is a function of community needs
+- To take action when reasonably confident that it is the right action to take
+- To fulfill the responsibilities outlined in this document and its dependents
+
+### Ratify Org Owners
+
+The [Ratify Org owners][ratify-project-owners] are responsible for:
+
+- Maintaining the mission, vision, values, and scope of the project
+- Refining the governance and charter as needed
+- Making project level decisions
+- Resolving escalated project decisions when the subproject owners responsible are blocked
+- Managing the Ratify brand
+- Controlling access to Ratify assets such as source repositories, hosting, project calendars
+- Deciding what subprojects are part of the Ratify project
+- Deciding on the creation of new subprojects
+- Overseeing the resolution and disclosure of security issues
+- Managing financial decisions related to the project
+
+Changes to org owners use the following:
+
+- Any subproject owner is eligible for a position as an org owner
+- No one company or organization can employ a simple majority of the org owners
+- An org owner may step down by submitting an [issue][ratify-issues-new] stating their intent and they will be moved to emeritus.
+- Org owners MUST remain active on the project. If they are unresponsive for > 6 months they will lose org ownership unless a [super-majority][super-majority] of the other org owners agree to extend the period to be greater than 6 months
+- When there is an opening for a new org owner, any current owners may nominate a suitable subproject owner as a replacement
+  - Nominations for new owners must be made by creating an [issue][ratify-issues-new].
+- When nominated individual(s) agrees to be a candidate for ownership, the subproject owners may vote
+  - The voting period will be open for a minimum of three business days and will remain open until a super-majority of project owners has voted
```

Comment:
```suggestion
  - The voting period will be open for a minimum of three business days and will remain open until a [two-thirds supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of project owners has voted
```

Reply:
[two-thirds supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) might be a more accurate definition to the "super-majority"